From: chrysn <chrysn@fsfe.org>
Date: Tue, 11 Aug 2015 09:29:40 +0200
Subject: man page: Rephase colorscheme option to avoid problems with wrapping

The old version threw errors when formatted:

$ LC_ALL=en_US.UTF-8 MANROFFSEQ='' MANWIDTH=80 man --warnings -E UTF-8 -l -Tutf8 -Z ./doc/openscad.1 >/dev/null
<standard input>:85: warning [p 2, 4.7i, div `an-div', 0.0i]: cannot adjust line

This was discovered by lintian and reported as
'manpage-has-errors-from-man'.

Forwarded: https://github.com/openscad/openscad/pull/1411
---
 doc/openscad.1 | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/doc/openscad.1 b/doc/openscad.1
index 4616a37..0ee66b0 100644
--- a/doc/openscad.1
+++ b/doc/openscad.1
@@ -82,8 +82,10 @@ If exporting an image, specify the pixel width and height
 If exporting an image, specify whether to use orthographic or perspective 
 projection
 .TP
-.B \-\-colorscheme=[Cornfield|Sunset|Metallic|Starnight|BeforeDawn|Nature|DeepOcean]
+.B \-\-colorscheme=\fIscheme
 If exporting an image, use the specified color scheme for the rendering.
+\fIscheme\fP can be any of \fBCornfield\fP, \fBSunset\fP, \fBMetallic\fP,
+\fBStarnight\fP, \fBBeforeDawn\fP, \fBNature\fP or \fBDeepOcean\fP.
 .TP
 .B \-v, \-\-version
 Show version of program.
